config/routes.rb

```diff
@@ -94,6 +94,17 @@
         end
       end
 
+      resources :exam_templates do
+        member do
+          get 'download'
+        end
+
+        collection do
+          get 'index'
```

Comment:
You need to review http://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default. Don't create routes that are already defined.